flink-dist/pom.xml

```diff
@@ -715,6 +715,8 @@ under the License.
 									<artifact>*</artifact>
 									<excludes>
 										<exclude>org/apache/flink/runtime/util/bash/BashJavaUtils.class</exclude>
+										<exclude>io/fabric8/**</exclude>
```

Comment:
The files of `io.fabric8` directory are `zjsonpatch` and `kubernetes`.
I think I agree with you before I add this exclude option. But when I removed this line in pom from flink-dist and package the whole project in my env. The `io.fabric8` actually is packaged into the flink-dist.jar .  The same behavior with the following line for "META-INF/services"(it will contain the shaded service file and non-shaded ones). If we remove both lines in flink-dist pom, the packaged flink-dist.jar will contain `io.fabric8` and `org.apache.flink.kubernetes.shaded.io.fabric8`, which contains the same class files. But the flink-kubernetes will package as wish. So I think the behavior might be related with the shade of flink-dist.

Reply:
My bad, this is mvn version dismatch. Will refresh this part. Thank you.